### PR TITLE
add basic ui-routing without auth and passport yet

### PR DESCRIPTION
**Pull request by slava89**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -28,18 +28,20 @@
...
-  <div>
-    <h1>Welcome to Best friends social network!!!!</h1>
-  </div>
-</section>
+<div ui-view></div>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.6/angular.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.3.1/angular-ui-router.js"></script>
+    <script src="/index.js"></script>
+    <script src="configs/states.js"></script>
```
Comment (andrewdacenko) [inline client/index.html]: Append slash at src

**Comment by slava89** [comment]
you mean at configs?

**Comment by slava89** [comment]
or you mean remove https?

**Comment by andrewdacenko** [comment]
@slava89 Both. /configs and remove https

**Pull request merged**